README.md

```diff
@@ -114,12 +114,13 @@ between client-python versions.
 | 7.0 Alpha/Beta  | Kubernetes main repo, 1.11 branch    | ✗                             |
 | 7.0             | Kubernetes main repo, 1.11 branch    | ✗                             |
 | 8.0 Alpha/Beta  | Kubernetes main repo, 1.12 branch    | ✗                             |
-| 8.0             | Kubernetes main repo, 1.12 branch    | ✓                             |
+| 8.0             | Kubernetes main repo, 1.12 branch    | ✗                             |
 | 9.0 Alpha/Beta  | Kubernetes main repo, 1.13 branch    | ✗                             |
 | 9.0             | Kubernetes main repo, 1.13 branch    | ✓                             |
 | 10.0 Alpha/Beta | Kubernetes main repo, 1.14 branch    | ✗                             |
 | 10.0            | Kubernetes main repo, 1.14 branch    | ✓                             |
 | 11.0 Alpha/Beta | Kubernetes main repo, 1.15 branch    | ✓                             |
```

Comment:
When the stable version is released, the alpha/beta support reaches its end.

Reply:
I think I missed that. Let me change it.

Reply:
Updated.